DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests.cs

```diff
@@ -132,8 +132,25 @@ public void AddRegistrationPrompt_journey_has_no_accessibility_errors()
             var selectPromptForm = driver.FindElement(By.TagName("form"));
             selectPromptForm.Submit();
 

```

Comment:
Nice! I wonder whether it's time to separate the journeys into different files.. 🤔 
So we could have AccessibilityTests folder and then BasicAccessibilityTests.cs, RegistrationJourneyAccessibilityTests.cs, AddCentreCustomPromptsAccessibilityTests.cs?
We could also create some helpers for logging in, submitting the form, filling in a text input, dropdown etc to make the tests nicer to read :)